server/app/services/geo/esri/EsriServiceAreaValidationOption.java

```diff
@@ -50,13 +51,14 @@ public abstract static class Builder {
 
   /**
    * Determine if this EsriServiceAreaValidationOption is in the provided list of {@link
-   * ServiceAreaInclusion}.
+   * ServiceAreaInclusion}. Filters out FAILED states.
```

Comment:
**suggestion (blocking):** more directly state the impact of this: `FAILED` inclusion checks return what value?

Reply:
Updated explanation of what an inclusion means to have a FAILED state and why it's getting filtered out.